src/components/Firebase/firebase.js

```diff
@@ -27,7 +27,6 @@ class Firebase {
   }
 
   // Points of interest firestore database
-  poi = () => this.db.collection("poi").doc();
```

Comment:
i'd say this is not the best of practices, that is, deleting things in an unrelated branch